etna/datasets/tsdataset.py

```diff
@@ -871,6 +898,12 @@ def index(self) -> pd.core.indexes.datetimes.DatetimeIndex:
         """
         return self.df.index
 
+    def level_names(self) -> Optional[List[str]]:
```

Comment:
Where is `get_level_segments`? According to the task #1028 it is planned to be added.

Reply:
I decided that it is useless here, we can we do we the segments of the different level than current in the dataset? Let's add it if we face the use case

Reply:
It was hard to understand due to typos)